src/components/counter/counter.js

```diff
@@ -0,0 +1,22 @@
+import CountUp from 'react-countup'
+
+import './counter.scss'
+
+function Counter() {
+  return (
+    <div className="count">
+      <CountUp
+        start={-98.039}
```

Comment:
If you have more than 1 counter component on your page, you should make it reusable by supply start, stop and other params as props.